.github/workflows/test.yml

```diff
@@ -163,7 +163,11 @@ jobs:
 
       - name: Destroy AWS stacks used for testing
         run: |
-          poetry run cdk destroy --force --all
+          cdk --app cdk.out ls | while read line ; do
```

Comment:
This lists the OIDC stack as well, which we don't want to destroy. You can avoid that by adding `'*-geostore'` to print only the Geostore stack.

Reply:
Have removed loop altogether to keep things simple for now. Happy to revisit should we introduce another stack into the mix in the future. Parsing through `cdk.out` is much faster than running `cdk ls`, though there is still some overhead (few seconds lead time) to get the actual stack names listed.